### ISSUE TITLE
Caching does not work for video files

### ISSUE THREAD

**Issue by xdenser** [nwjs/nw.js#1419]
Given following code
``` html
<!doctype HTML>
<html>
  <body>	
    <video style="left:0px;top: 0px;" loop autoplay>
          <source src='http://video.webmfiles.org/big-buck-bunny_trailer.webm'>
     </video>
  </body>	
</html>
```
in chrome dev tools
![chrome](https://f.cloud.github.com/assets/523984/1808478/7249a9e2-6d56-11e3-9b4b-3f78093a2c96.png)

in nw dev tools
![nw](https://f.cloud.github.com/assets/523984/1808479/7abcfa5c-6d56-11e3-8a73-095c9f79f66a.png)

so video is reloaded on each loop
in chrome after page reload it is also taken from cache

tested in 0.7.5, 0.8.3, 0.9.1 on  Win7 x64

**Comment by manast** [comment]
+1

**Comment by ghostoy** [comment]
The cache backend is changed in 0.11.6. Could you check if it fixes your issue?

**Comment by bluelightzero** [comment]
I'm afraid I don't have access to the code at the moment, but it was simple
to replicate.

Have any video tag in the document which takes time to load, and trace /
plot on canvas the percentage loaded using JavaScript.

My time is consumed with other things right now, but I plan to look at this
again in the future.

Thank you for the fix.

**Comment by ghostoy** [comment]
I think it's fixed in 0.11.6. See screenshot.
![screen shot 2015-01-23 at 10 29 40 am](https://cloud.githubusercontent.com/assets/471928/5868870/091066fc-a2eb-11e4-97e8-d58179778928.png)
